sdk/keyvault/keyvault-admin/review/keyvault-admin.api.md

```diff
@@ -89,20 +100,25 @@ export class KeyVaultBackupClient {
     readonly vaultUrl: string;
 }
 
+// Warning: (ae-forgotten-export) The symbol "DataAction" needs to be exported by the entry point index.d.ts
+//
+// @public
+export type KeyVaultDataAction = DataAction;
```

Comment:
What's the best way to handle this scenario? You can see that DataAction is defined here: https://github.com/Azure/azure-sdk-for-js/blob/b39083d2dd977d55ca97efd8fb682e36b35a816c/sdk/keyvault/keyvault-admin/src/generated/models/index.ts#L405-L434

I will add doc comment but I'm not sure what the right thing to do here is? I noticed we don't export the generated types directly so figured I'll ask what's the right way to handle that

Reply:
@maorleger we end up copying the types from the generated folder out to the convenience layer. The idea is that in the cases in which the generated types change, we should still be able to avoid changing our public API types.

Reply:
Thanks so to confirm in this case you'd recommend copying the type over including duplicating all of the values, right?

Reply:
Yes, I would recommend that, 100%

Reply:
I mean duplicating all of the values! Check what we do in keys. I mean, they're not exactly values, right? I was confused for a bit. They're like, string literals? some non-value type 😁 anyway